Dockerfile

```diff
@@ -48,6 +48,9 @@ RUN apt-get -y install \
     latex-cjk-chinese-arphic-gbsn00lp latex-cjk-chinese-arphic-gkai00mp \
     fonts-symbola
 
+# Install 'rsvg-convert' for SVG -> PDF conversion
+RUN apt-get -y install librsvg2-bin
```

Comment:
I would add a comment here with this link also, https://github.com/missinglinkelectronics/sphinxcontrib-svg2pdfconverter

Reply:
Done: 80f52ba4130c271f7b1fa94ff6e593b2d6ce0614